src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersion.cs

```diff
@@ -76,11 +76,6 @@ public class DataSetVersion : ICreatedUpdatedTimestamps<DateTimeOffset, DateTime
     public DataSetVersionType VersionType
         => VersionMinor == 0 ? DataSetVersionType.Major : DataSetVersionType.Minor;
 
-    public bool CanBeDeleted => Status is DataSetVersionStatus.Failed
-        or DataSetVersionStatus.Mapping
-        or DataSetVersionStatus.Draft
-        or DataSetVersionStatus.Cancelled;
-
```

Comment:
Similar to the previous PR, would keep this in the model - not sure why this was moved into an extension

Reply:
I've addressed this in [EES-5173](https://github.com/dfe-analytical-services/explore-education-statistics/pull/4957)